src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -262,27 +262,31 @@
    environment :- schema/Str
    jruby-service :- (schema/protocol jruby-protocol/JRubyPuppetService)
    request-tag :- (schema/maybe String)
-   last-updated :- (schema/maybe schema/Int)]
+   last-updated :- (schema/maybe schema/Int)
+   environment-class-cache-enabled :- schema/Bool]
```

Comment:
Yeah, that's a good idea, thanks.  I didn't have anything about the other parameters in the docstring either but that may be helpful to include as well.  I'll keep that in mind as I rework the code for #930.